src/test/resources/org/mapfish/print/config/sample_config_yaml/config-georchestra.yaml

```diff
@@ -48,9 +48,9 @@ brokenUrlPlaceholder: http://upload.wikimedia.org/wikipedia/commons/c/ce/Transpa
 security:
   - !basicAuth
     matcher: !dnsMatch
-      host: @shared.server.name@
-    username: @shared.privileged.geoserver.user@
-    password: @shared.privileged.geoserver.pass@
+      host: shared.server.name@
+    username: shared.privileged.geoserver.user
+    password: shared.privileged.geoserver.pass
```

Comment:
Why we removed the `@` characters here? Is there a specific reason?

Reply:
Yes, YAML does not allow that. This is one of the examples of the malformed yaml, where JYaml would simply ignore invalid properties, but the new library throws an exception